### PR TITLE
✨ wizard: add tag field in garden, fix license options

### PR DESCRIPTION
**Pull request by lucasrodes**
- [x] Fixes license selectbox (removes invalid options)
- [x] Adds field to add tag to indicators
- [ ] Adds missing fields in dummy template

Test it by using `etl-wizard garden`

### CODE REVIEW

File: apps/wizard/templating/cookiecutter/garden/{{cookiecutter.namespace}}/{{cookiecutter.version}}/{{cookiecutter.short_name}}.meta.yml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 # NOTE: To learn more about the fields, hover over their names.
+{%- if cookiecutter.topic_tags %}
+definitions:
+  commons:
```
Comment (paarriagadap) [inline apps/wizard/templating/cookiecutter/garden/{{cookiecutter.namespace}}/{{cookiecutter.version}}/{{cookiecutter.short_name}}.meta.yml:4]: `common`

**Review by paarriagadap — CHANGES_REQUESTED** [review]
Hi @lucasrodes!

I ran it and I could only see that the failing dropdown option in snapshot is gone and that the tags are saved in the yaml file.

- Missing fields are still missing in the dummy structure, like `variable.description_processing`
- It is adding `commons` instead of `common`

**Comment by lucasrodes** [comment]
@paarriagadap thanks for your feedback, I think I've implemented the changes that you suggested!

**Comment by paarriagadap** [comment]
Thanks @lucasrodes ! I don't see the new commits though

**Comment by lucasrodes** [comment]
@paarriagadap sorry, now!

**Review by paarriagadap — APPROVED** [review]
Good! I see there are some parameters missing from the dummy structure, but they are seldom used:
variable.display.shortUnit
variable.display.unit
variable.display.tableDisplay.hideAbsoluteChange
variable.display.tableDisplay.hideRelativeChange

Feel free to add them

**Pull request merged**